pages/glossary/programmatically-determined-context.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Programmatically determined context
+key: programmatically-determined-context
+---
+
+The programmatically determined context of a piece of information (_e.g._ an element) is all the additional information that has a [programmatically determinable](https://www.w3.org/TR/WCAG21/#dfn-programmatically-determinable) [relationship](https://www.w3.org/TR/WCAG21/#dfn-relationships) to it.
+
+It includes:
+
+- the element's [accessible name][];
```

Comment:
For a generic definition of programmatically determined context, I would argue that the accessible name is content (that can be determined programmatically). For this rule, after re-reading the SC, I have to agree that it should not be in the definition of programmatically determined **link** context. Consequently:

- I will move and update the current definition programmatically-determined-link-context in #693 to this PR so that we can move on (since #693 is on hold)
- I will move this definition of programmatically-determined-context to #905 which was waiting for this PR for such a definition

Reply:
#905 is closed so I will keep both definitions here in the meantime

Reply:
I'll remove it from this PR and keep it for future needs.